types/types.go

```diff
@@ -204,6 +204,9 @@ const (
 
 	CniNetworkNone          = ""
 	StorageNetworkInterface = "lhnet1"
+
+	KubeApiQps   = 50
+	KubeApiBurst = 100
```

Comment:
Just curious what will happen if using bigger numbers.

Reply:
I don't see benefit from increasing it to bigger values when testing scaling yet. These values seems big enough to eliminate the throttling